### DISCUSSION TITLE
SVG-Licenses

### DISCUSSION THREAD

**Discussion by tutralex** [hardinfo2/hardinfo2#94] [General]
**Deleted**:
 - printer2SVG.pdf - checked, not used;
 - wirelessSVG.pdf - was drawn by me, this license is not required;
 - nqueensSVG.pdf - was drawn by me, this license is not required;
 - networkSVG.pdf - was drawn by me, this license is not required;
 - mouseSVG.pdf - was drawn by me, this license is not required;
 - audioSVG.pdf - was drawn by me, this license is not required;
 - batterySVG.pdf - was drawn by me, this license is not required;
 - compressSVG.pdf - was drawn by me, this license is not required;
 - network-arpSVG.pdf - was drawn by me, this license is not required;
 - network-connections.pdf - was drawn by me, this license is not required;
 - dmiSVG.pdf - was drawn by me, this license is not required;
 - processorSVG.pdf - the license has been changed to the one that the current "processor.svg" is based on.

Licenses for other SVG-images, which are used in their original form or have been partially modified, are present in the "LICENSES" folder.

All other SVG-images I've drawn for the Hardinfo2 project are automatically licensed under the CC0 license in the "new_svg_icons.txt" file. The line "pixmaps/processor.svg" should be excluded from the file "new_svg_icons.txt".

Perhaps a license should be added for "mbSVG.pdf" since that image was the basis for "mb.svg".
[mbSVG.pdf](https://github.com/user-attachments/files/17497014/mbSVG.pdf)

**Comment by hwspeedy** [comment]
Good job @tutralex!

Checked the mentioned licenses changes - Seams all correct.

processorSVG.pdf is the correct one.

Lets also note all your SVG creationgs in new_svg_icons.txt - it's an impressive work and needs mentioning in license file and also - so we do not have to traverse git to check image licenses.

Please update new_svg_icons.txt and add mbSVG.pdf and remove all the *SVG.pdf, where you made new ones - in a single commit, thanx.

**Reply by tutralex** [reply]
Before I make the final changes to the licenses, I would like to suggest for consideration the image for the devices "devices.svg". I also slightly modified the image "network.svg" to more clearly draw the connections. The changes are minor and will not affect the release.
![devices](https://github.com/user-attachments/assets/3433209f-276a-456a-8123-83dd57c3ea39)
![network](https://github.com/user-attachments/assets/c392a183-d014-4a62-bb19-75a832456b7e)

**Comment by hwspeedy** [comment]
@tutralex 
They are very nice and accepted!

**Just do the right way**
1. Submit the first agreed commit: "LICENSE SVG updated to reflect current svg icons". Still having devicesSVG.pdf.
2. Then submit a second commit: "FIX Updated network.svg and new devices.svg" along with removal of devicesSVG.pdf and addition to new_svg_icons.txt.

Then the repo licenses are always in sync and we do not mix submits.
I know - these 2 commits could be merged, but the above is the correct way - so please try it, thanx
